plugins/modules/global_parameter.py

```diff
@@ -132,7 +145,7 @@ def main():
     )
 
     with module.api_connection():
-        entity = module.lookup_entity('entity')
+        entity = module.lookup_entity('entity', params={'show_hidden': True})
```

Comment:
smart_class_parameter also has support for `hidden_value`, but doesn't pass `show_hidden`, is that a bug (in smart_class_parameter)?

Reply:
That probably means that we do not yet support hidden values for smart_class_parameter.
We are also still missing that feature for nested parameter in host, domain, you-name-it.

Reply:
not support them in an idempotent way? but yeah.